### PR TITLE
Renames llama-pack => BubbleWrap

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +37,6 @@ See [CONTRIBUTING](./CONTRIBUTING.md) for more.
 
 See [LICENSE](./LICENSE) for more.
 
-## Disclaimer
+## Disclaimer
```
Comment (rayankans) [inline README.md]: remove trailing space

File: packages/cli/README.md [hunk 2]
```diff
@@ -96,6 +92,6 @@ See [CONTRIBUTING](../../CONTRIBUTING.md) for more.
 
 See [LICENSE](../../LICENSE) for more.
 
-## Disclaimer
+## Disclaimer
```
Comment (rayankans) [inline packages/cli/README.md]: remove trailing space

File: packages/core/README.md [hunk 3]
```diff
@@ -61,6 +57,7 @@ See [CONTRIBUTING](../../CONTRIBUTING.md) for more.
 
 See [LICENSE](../../LICENSE) for more.
 
-## Disclaimer
+## Disclaimer
```
Comment (rayankans) [inline packages/core/README.md]: trailing space

**Comment by andreban** [comment]
Fixed the trailing spaces

**Pull request merged**